src/modules/ProxiedModuleApi.ts

```diff
@@ -81,21 +82,21 @@ export class ProxiedModuleApi implements ModuleApi {
      * @override
      */
     public openDialog<M extends object, P extends DialogProps, C extends DialogContent<P>>(
-        title: string,
+        titleOrOptions: string | ModuleUiDialogOptions,
         body: (props: P, ref: React.RefObject<C>) => React.ReactNode,
         props?: Omit<P, keyof DialogProps>,
     ): Promise<{ didOkOrSubmit: boolean; model: M }> {
+        const options: ModuleUiDialogOptions =
+            typeof titleOrOptions === "string" ? { title: titleOrOptions } : titleOrOptions;
+
         return new Promise<{ didOkOrSubmit: boolean; model: M }>((resolve) => {
             Modal.createDialog(
                 ModuleUiDialog<P, C>,
                 {
-                    title: title,
+                    ...options,
```

Comment:
I feel like it might be cleaner to pass in the whole `options` object as a property, rather than duplicating each of its properties into `IProps`.

```suggestion
                    initialOptions: options,
```

what do you think?

Reply:
I like it!